tmt/steps/finish/shell.py

```diff
@@ -33,11 +33,11 @@ class FinishShell(tmt.steps.finish.FinishPlugin):  # type: ignore[misc]
     @classmethod
     def options(cls, how: Optional[str] = None) -> Any:
         """ Finish command line options """
-        return [
+        return cast(List[tmt.options.ClickOptionDecoratorType], [
```

Comment:
please look @ https://github.com/teemtee/tmt/pull/1418/files#diff-869f5ba1150a929f89989f134e4591435b5c0a66e2cf553a046357239a135b7eR57

and similar how we agreed to type this in other modules

Reply:
Changed. Thanks.